models/files/xgbDART.R

```diff
@@ -0,0 +1,262 @@
+#xgbDART 
+modelInfo <- list(label = "eXtreme Gradient Boosting",
+                  library = c("xgboost", "plyr"),
+                  check = function(pkg) {
+                    requireNamespace("xgboost")
+                    current <- packageDescription("xgboost")$Version
+                    expected <- "0.6.4"
+                    if(compareVersion(current, expected) < 0)
+                      stop("This modeling workflow requires xgboost version ",
+                           expected, " or greater. Consider using the drat repo.", call. = FALSE)
+                  },
+                  type = c("Regression", "Classification"),
+                  parameters = data.frame(parameter = c("nrounds", 
+                                                        "max_depth",
+                                                        "eta",
+                                                        "gamma",
+                                                        "subsample",
+                                                        "colsample_bytree",
+                                                        "rate_drop",
+                                                        "skip_drop", 
+                                                        "min_child_weight"),
+                                          class = c(rep("numeric", 9)),
+                                          label = c("# Boosting Iterations", 
+                                                    "Max Tree Depth",
+                                                    "Shrinkage",
+                                                    "Minimum Loss Reduction",
+                                                    "Subsample Percentage",
+                                                    "Subsample Ratio of Columns",
+                                                    "Fraction of previous trees to drop during dropout",
```

Comment:
Can you shorten these and use consistent capitalization (e.g. maybe "Fraction of Previous Trees")? The labels might get used in `ggplot` legends or facets and long labels might be an issue.